### ISSUE TITLE
Produce jssm-viz on deno-land

### ISSUE THREAD

**Issue by StoneCypher** [StoneCypher/fsl#916]

**Comment by StoneCypher** [comment]
Maybe impossible, maybe nevermind 😦 

Blocked by #921

**Comment by StoneCypher** [comment]
Closing as not planned: jssm-viz was deprecated and folded into jssm at 5.105.0; there is no package to publish.

Dispositioned in the #1849 legacy-milestone sweep.
